pkg/networking/networking_test.go

```diff
@@ -30,10 +30,11 @@ func Test_GetDefaultHeader_WithAuth(t *testing.T) {
 		"Authorization": {"token " + token},
 	}
 
-	url, _ := url.Parse(config.GetString(configuration.API_URL))
-	actualHeader := net.GetDefaultHeader(url)
-
-	assert.Equal(t, expectedHeader, actualHeader)
+	for i := 0; i < 3; i++ {
```

Comment:
Why the magic number 3?

Reply:
I just wanted to run the method under test multiple times to ensure it behaves the same each time. 2 was to obvious, 3 felt reasonable. shall I add a comment?

Reply:
Yeah, I'd do that just to make it clear